### DISCUSSION TITLE
What about Photoprism updates?

### DISCUSSION THREAD

**Discussion by leofabri** [tteck/Proxmox#76] [Q&A] [answered]
Hello and thank you for the amazing work!

What if Photoprism releases a new version and I need to update it?

**Comment by tteck** [comment, marked as answer]
Or, if you wanted to switch to the develop branch
```yaml
bash -c "$(wget -qLO - https://raw.githubusercontent.com/tteck/Proxmox/main/misc/photoprism-update.sh)"
```

**Reply by Shadoweee77** [reply]
This is main Proxmox console script or rather the Photoprism LXC. Also will this keep old config as is?
Edit: I was tired :)

**Reply by tteck** [reply]
?? Jellyfin
The script above is to update PhotoPrism, run in the LXC Console. Yes it will keep the configuration.
I will refer to Proxmox Shell or LXC Console. Never Proxmox console or LXC shell

**Reply by automaton82** [reply]
@tteck it appears you deleted `photoprism-update.sh` with the comment prune. Is there no alternative anymore to update photo prism?

**Reply by tteck** [reply]
@automaton82 What does the instructions say?

**Reply by automaton82** [reply]
I don't know, I installed it via your script last year and put the update script in cron and now it's failing. Then I searched the error and ended up here.

Where are the instructions? https://github.com/tteck/Proxmox/blob/main/README.md doesn't say.

**Comment by leofabri** [comment]
I see, I'm gonna try that today. Thank you for the support @tteck
